pkg/controller/bootstrap/bootstrapsigner.go

```diff
@@ -191,10 +192,12 @@ func (e *Signer) signConfigMap(ctx context.Context) {
 
 	newCM := origCM.DeepCopy()
 
+	logger := klog.FromContext(ctx)
+
 	// First capture the config we are signing
 	content, ok := newCM.Data[bootstrapapi.KubeConfigKey]
 	if !ok {
-		klog.V(3).Infof("No %s key in %s/%s ConfigMap", bootstrapapi.KubeConfigKey, origCM.Namespace, origCM.Name)
+		logger.V(3).Info("No KubeConfigKey in ConfigMap", "KubeConfigKey", bootstrapapi.KubeConfigKey, "configMap", klog.KObj(origCM))
```

Comment:
/hold
verify ci failed.

Reply:
It looks like the `logtools` validation tool has been updated. I  have updated the PR for validation.